dotnet/SimpleSauce.Test/SauceOptionsTests.cs

```diff
@@ -19,49 +19,51 @@ public void Setup()
         [TestMethod]
         public void UsesLatestChromeWindowsVersionsByDefault()
         {
-            SauceOptions.BrowserName.Should().BeEquivalentTo(Browser.Chrome);
-            SauceOptions.BrowserVersion.Should().Be("latest");
-            SauceOptions.PlatformName.Should().BeEquivalentTo(Platforms.Windows10);
+            SauceOptions.ConfiguredOptions.BrowserName.Should().BeEquivalentTo(Browser.Chrome.Value);
+            SauceOptions.ConfiguredOptions.BrowserVersion.Should().Be("latest");
+            SauceOptions.ConfiguredOptions.PlatformName.Should().BeEquivalentTo(Platforms.Windows10.Value);
         }
+
         [TestMethod]
-        public void UpdatesBrowserBrowserVersionPlatformVersion()
+        public void UpdatesBrowserAndBrowserVersionAndPlatformVersion()
         {
-            SauceOptions.BrowserVersion = "68";
-            SauceOptions.BrowserName = Browser.Firefox;
-            SauceOptions.PlatformName = Platforms.MacOsHighSierra;
+            SauceOptions.ConfiguredOptions.BrowserVersion = "68";
+            //SauceOptions.BrowserName = Browser.Firefox.Value;
```

Comment:
Codacy found an issue: [Remove this commented out code.](https://app.codacy.com/manual/SauceLabs/sauce_bindings/pullRequest?prid=6342070)